packages/shared/src/components/sidebar/SidebarListItem.tsx

```diff
@@ -28,7 +28,7 @@ function SidebarListItem({
   );
   const content = (
     <>
-      {icon}
+      {React.cloneElement(icon as ReactElement, { secondary: isActive })}
```

Comment:
Why did you clone it here?

Reply:
To add the props to the render, basically because we have neater accessibility to the isActive.
Would you say it's neater to add it higher up but manually check with strings?

Reply:
Ahh, this is neater ❤️